jobs/e2e_node/image-config-serial.yaml

```diff
@@ -6,14 +6,14 @@ images:
     image: ubuntu-gke-1804-1-16-v20200330 # docker 17.03
     project: ubuntu-os-gke-cloud
   cos-stable2:
-    image: cos-77-12371-274-0 # docker v19.03.1, deprecated after 2020-12-17
+    image_family: cos-81-lts # docker v19.03.6, deprecated after 2021-06-24
```

Comment:
Last PR we removed the docker version from the comment as it doesn't affect anything. Can we do the same here?

Reply:
Sure, I can remove it in next patch. 
I do not see docker, or deprecation matters, since those info are changing and also listed in COS release notes